google/cloud/forseti/scanner/scanners/bucket_rules_scanner.py

```diff
@@ -121,8 +121,8 @@ def _retrieve(self):
         scoped_session, data_access = model_manager.get(self.model_name)
         with scoped_session as session:
             bucket_acls = []
-
-            for gcs_policy in data_access.scanner_iter(session, 'gcs_policy'):
+            policies = [policy for policy in data_access.scanner_iter(session, 'gcs_policy')]
+            for gcs_policy in policies:
```

Comment:
nit: can you please make the naming consistent?
change `policies` to `gcs_policies`

Reply:
Fixed!